tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -43,40 +45,72 @@ class ConfigurationPassImpl @Inject()(val options: PassOptions,
     * @return true, if the pass was successful
     */
   override def execute(): Boolean = {
-    var module = tlaModule.get
+    // try to read from the TLC configuration file
+    val configuredModule = tryConfigureFromTlcConfig(tlaModule.get)
 
+    // dump the configuration result
+    val outdir = options.getOrError("io", "outdir").asInstanceOf[Path]
+    PrettyWriter.write(configuredModule, new File(outdir.toFile, "out-config.tla"))
+
+    // set fallback options
+    setFallbackOptions()
+
+    // make sure that the operators are defined
+    ensureDeclarationsArePresent(configuredModule)
+
+    outputTlaModule = Some(configuredModule)
+    true
+  }
+
+  // if checker.init and checker.next are not set, set them to Init and Next, respectively
+  private def setFallbackOptions(): Unit = {
+    if (options.get("checker", "init").isEmpty) {
+      logger.info("  > Option --init is not set. Using Init")
+      options.set("checker.init", "Init")
+    }
+    if (options.get("checker", "next").isEmpty) {
+      logger.info("  > Option --next is not set. Using Next")
```

Comment:
Suggested for the same reason as above:

```suggestion
      logger.debug("  > Option --next is not set. Using Next")
```